pkg/util/rolesets/const.go

```diff
@@ -0,0 +1,84 @@
+package rolesets
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+var DefaultPlatformWorkloadIdentityRoleSet = api.PlatformWorkloadIdentityRoleSet{
```

Comment:
Should it be a Go embed or an environment variable in the secrets? I mostly copied the approach from the way OCP versions handles it now, but I'm definitely open to a better idea.

Reply:
Since the role sets are stored in RP-Config but we don't have RP-Config in dev, I think it would have to be sourced in the secrets or left as-is.

Reply:
In the long term, a Go embed sounds nice because you'd be able to copy-paste the one you want to use in your dev environment from RP-Config into the file and then have the script read it in. But I feel like it would make sense to leave that as a refactor for later so that it can be done to this script and the OCP versions script at the same time.

Reply:
Agreed, I'm going to resolve this comment with justification that any refactor here should also include the openShiftVersions script (since this is mostly copied over from that) and is something that we could tackle separately.

Reply:
This could be a `good-first-item` jira to have in the backlog. Has it been carded?